modules/rhcos-enabling-multipath.adoc

```diff
@@ -45,53 +45,60 @@ The following procedure enables multipath at installation time and appends kerne
 ----
 $ mpathconf --enable && systemctl start multipathd.service
 ----
-** Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
++
+.. Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
 
 . Append the kernel arguments by invoking the `coreos-installer` program:
 +
 * If there is only one multipath device connected to the machine, it should be available at path `/dev/mapper/mpatha`. For example:
 +
+ifndef::restricted[]
 [source,terminal]
 ----
-ifndef::restricted[]
-$ coreos-installer install /dev/mapper/mpatha \// <1>
+$ coreos-installer install /dev/mapper/mpatha \//
 --ignition-url=http://host/worker.ign \
 --append-karg rd.multipath=default \
 --append-karg root=/dev/disk/by-label/dm-mpath-root \
 --append-karg rw
+----
 endif::[]
 ifdef::restricted[]
-$ coreos-installer install /dev/mapper/mpatha \// <1>
+[source,terminal]
+----
+$ coreos-installer install /dev/mapper/mpatha \//
 --ignition-url=http://host/worker.ign \
 --append-karg rd.multipath=default \
 --append-karg root=/dev/disk/by-label/dm-mpath-root \
 --append-karg rw \
 --offline
-endif::[]
 ----
-<1> Indicates the path of the single multipathed device.
+endif::[]
+* `/dev/mapper/mpatha`: Indicates the path of the single multipathed device.
 +
 * If there are multiple multipath devices connected to the machine, or to be more explicit, instead of using `/dev/mapper/mpatha`, it is recommended to use the World Wide Name (WWN) symlink available in `/dev/disk/by-id`. For example:
 +
+ifndef::restricted[]
 [source,terminal]
 ----
-ifndef::restricted[]
 $ coreos-installer install /dev/disk/by-id/wwn-<wwn_ID> \// <1>
 --ignition-url=http://host/worker.ign \
 --append-karg rd.multipath=default \
 --append-karg root=/dev/disk/by-label/dm-mpath-root \
 --append-karg rw
+----
 endif::[]
 ifdef::restricted[]
+[source,terminal]
+----
 $ coreos-installer install /dev/disk/by-id/wwn-<wwn_ID> \// <1>
 --ignition-url=http://host/worker.ign \
 --append-karg rd.multipath=default \
 --append-karg root=/dev/disk/by-label/dm-mpath-root \
 --append-karg rw \
 --offline
-endif::[]
 ----
-<1> Indicates the WWN ID of the target multipathed device. For example, `0xx194e957fcedb4841`.
+endif::[]
+* `<wwn_ID>`: Indicates the WWN ID of the target multipathed device. For example, `0xx194e957fcedb4841`.
```

Comment:
Double-check the formatting here:
<img width="727" height="414" alt="Screenshot 2025-11-14 at 2 26 49 PM" src="https://github.com/user-attachments/assets/0ff38245-104c-4e3d-82f9-e909a0069c8a" />